webapp/src/blocks.tsx

```diff
@@ -1005,7 +1009,7 @@ export class Editor extends toolboxeditor.ToolboxEditor {
 
     display(): JSX.Element {
         let flyoutOnly = this.parent.state.editorState && this.parent.state.editorState.hasCategories === false;
-        let showErrorList = pxt.Util.isFeatureEnabled("blocksErrorList");
+        let showErrorList = pxt.Util.isFeatureEnabled("blocksErrorList") && !pxt.shell.isTimeMachineEmbed() && !this.parent.state.debugging;
```

Comment:
I'm wondering if this should also just be for `isReadOnly`..  Although, I think it might be helpful for the code eval tool. I'm just not sure if it would be nice to have in our other read only modes.

![image](https://github.com/user-attachments/assets/e6a47ddf-e8b2-4eb8-82f6-7690e7d0aa0b)